lib/pbench/server/auth/__init__.py

```diff
@@ -397,15 +399,15 @@ def _method(
             raise OpenIDCAuthenticationError(response.status_code)
         except (requests.exceptions.ConnectionError, requests.exceptions.Timeout) as e:
             self.logger.exception(
-                f"Could not connect to the OIDC client {self.__repr__()}"
+                "Could not connect to the OIDC client {}", self.__repr__()
```

Comment:
`repr(self)` would look a bit more conventional than directly referencing the `__repr__()` method. However, you could also use `"{!r}", self`.